src/api/worker.js

```diff
@@ -8,5 +8,48 @@ export default {
   async get() {
     const res = await request.get('workers')
     return await res.json()
-  }
+  },
+
+  /**
+   * Get worker
+   * @param {Number} id
+   * @return {Promise}
+   */
+  async get(id) {
+    const res = await request.get(`workers/${id}`)
+    return await res.json()
+  },
+
+  /**
+   * Create worker
+   * @param {Object} username and password
+   * @return {Promise}
+   */
+   async create(username,password) {
```

Comment:
Change argument to user
```suggestion
   async create(worker) {
```

Reply:
We need to validate data before it's passed on to the request.

Reply:
I've created a validation library once, it could be used to ease validation if we want a simple solution. 
[@specla/validator](https://github.com/specla/validator)

Reply:
Another alternative is [superstruct](https://github.com/ianstormtaylor/superstruct).